bindings/matrix-sdk-ffi/Cargo.toml

```
@@ -46,6 +46,7 @@ url = { workspace = true }
 zeroize = { workspace = true }
 uuid = { version = "1.4.1", features = ["v4"] }
 language-tags = "0.3.2"
+serde_html_form.workspace = true
```

Please be consistent with the rest of the `Cargo.toml` file 🙂:

```suggestion
serde_html_form = { workspace = true }
```

---

Oh, sorry, I didn't realize the format was different elsewhere. I blindly ran `cargo add`. 🙈